### PR TITLE
pep8, minor changes

### PR DESCRIPTION
**Pull request by whoburg**
@bqpd, please review. Ready for merging.

### CODE REVIEW

**Comment by whoburg** [comment]
(fails Jenkins because `newconstr_and_keydict` fails Jenkins; should not prevent merging)

File: gpkit/tests/t_sub.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def test_numeric(self):
         x = Variable("x")
         p = x**2
         self.assertEqual(p.sub(x, 3), 9)
-        self.assertEqual(p.sub(x.varkeys["x"][0], 3), 9)
+        self.assertEqual(p.sub(x.key, 3), 9)
```
Comment (bqpd) [inline gpkit/tests/t_sub.py]: Should work with just `sub(x, 3)`

**Comment by bqpd** [comment]
One comment, otherwise looks great!

File: gpkit/tests/t_sub.py (hunk 1 above)
Comment (bqpd) [inline gpkit/tests/t_sub.py]: oh, lol, that's done above. Gotcha. Merging!

**Pull request merged**